nemo/collections/nlp/models/language_modeling/lm_model.py

```diff
@@ -35,7 +37,7 @@
 @experimental
 class BERTLMModel(ModelPT):
```

Comment:
A minor comment, but if I were writing this class, I would call it BERTMLMModel - because it is masked language modeling task, not typical (autoregressive) LM. Up to you how you want to call it though.

Reply:
@yzhang123 , what do you want to call this model? I Agree BERTMLMModel is better.

Reply:
I changed to BERTMLMModel